src/closer-core.coffee

```diff
@@ -700,6 +700,26 @@ core =
         core.group_$_by(core.identity, coll))
      )
 
+  'not_$_every_$QMARK_' : (pred,coll) ->
+    assertions.arity 2, arguments.length
+    assertions.function pred
+    assertions.collection coll
+    !m.every? pred,coll
+    
+  'not_$_any_$QMARK_' : (pred,coll) ->
+    assertions.arity 2, arguments.length
+    assertions.function pred
+    assertions.collection coll
+    !m.some pred, coll
+  
+  'distinct_$QMARK_' : (args...) ->
+    assertions.arity 1, Infinity, arguments.length
+    if arguments.length > m.count(m.set args)
+      false
+    else
+      true
```

Comment:
Since you're testing a condition and returning a boolean, you can directly say `return arguments.length is m.count(m.set args)`.